docs/topics/signals.txt

```diff
@@ -190,6 +190,14 @@ In some circumstances, the code connecting receivers to signals may run
 multiple times. This can cause your receiver function to be registered more
 than once, and thus called multiple times for a single signal event.
 
+Although :meth:`~django.apps.AppConfig.ready` is designed to be called
+only once during Django's startup sequence, unit test runs may break this
+implicit contract. Another (albeit less likely) cause is if you import the
+module where you define the signal(s) from somewhere in your project;
+the signal registration will run as many times as it is imported.
+As projects tend to grow larger over time (more features, bug fixes),
+this issue could present itself.
```

Comment:
I would chop this.

Reply:
Yeah, it's a "software engineering" honest moment, but agreed, it doesn't add anything explicit or necessary.